Description: Some improvements in options info
Author: Agustin Martin Domingo <agmartin@debian.org>
Forwarded: http://bugzilla.abisource.com/show_bug.cgi?id=13163
Bug: http://bugzilla.abisource.com/show_bug.cgi?id=13163
Bug-Debian: http://bugs.debian.org/546418
Bug-Debian: http://bugs.debian.org/647281

* Improve wording for -a option.
* Add info about -d option to man page.
* Add info about -h option to man page and usage.

Index: enchant/doc/enchant.1
===================================================================
--- enchant.orig/doc/enchant.1	2011-10-29 08:49:27.000000000 +0200
+++ enchant/doc/enchant.1	2011-10-29 08:51:46.000000000 +0200
@@ -5,7 +5,7 @@
 .SH SYNOPSIS
 .ll +8
 .B enchant
-[\fB\-a\fR] [\fB\-l\fR] [\fB\-L\fR] [\fB\-v\fR]
+[\fB\-a\fR] [\fB\-d dict\fR] [\fB\-h\fR] [\fB\-l\fR] [\fB\-L\fR] [\fB\-v\fR]
 .ll -8
 .br
 .SH DESCRIPTION
@@ -14,7 +14,13 @@
 .SS OPTIONS
 .TP
 .B "\-a"
-List alternatives.
+List suggestions in ispell pipe mode format.
+.TP
+.B "\-d dict"
+Use dictionary <dict>.
+.TP
+.B "\-h"
+Show short help.
 .TP
 .B "\-l"
 List only the misspellings.
Index: enchant/tests/enchant-ispell.c
===================================================================
--- enchant.orig/tests/enchant-ispell.c	2011-10-29 08:49:13.000000000 +0200
+++ enchant/tests/enchant-ispell.c	2011-10-29 08:52:29.000000000 +0200
@@ -71,8 +71,9 @@
 print_help (FILE * to, const char * prog)
 {
 	fprintf (to, "Usage: %s [options] -a|-d dict|-l|-L|-m|-v[v]|<file>\n", prog);
-	fprintf (to, "\t-a lists alternatives.\n");
+	fprintf (to, "\t-a lists suggestions in ispell pipe mode format.\n");
 	fprintf (to, "\t-d dict uses dictionary <dict>.\n");
+	fprintf (to, "\t-h Show this help message.\n");
 	fprintf (to, "\t-l lists misspellings.\n");
 	fprintf (to, "\t-m is ignored.\n");
 	fprintf (to, "\t-L displays line numbers.\n");
